src/plugins/echo_plugin/echo_plugin.cc

```diff
@@ -101,13 +96,12 @@ bool EchoPlugin::Initialize(runtime::core::AimRTCore* core_ptr) noexcept {
     AIMRT_TRACE("Load {} pkg and {} type.",
                 type_support_pkg_loader_vec_.size(), type_support_map_.size());
 
-    RegisterGetTypeSupportFunc(
-        [this](std::string_view msg_type) -> aimrt::util::TypeSupportRef {
-          auto finditr = type_support_map_.find(msg_type);
-          if (finditr != type_support_map_.end())
-            return finditr->second.type_support_ref;
-          return {};
-        });
+    get_type_support_func_ = [this](std::string_view msg_type) -> aimrt::util::TypeSupportRef {
+      auto finditr = type_support_map_.find(msg_type);
+      if (finditr != type_support_map_.end())
+        return finditr->second.type_support_ref;
+      return {};
+    };
```

Comment:
This lambda doesn't appear to capture any additional data as a closure.  Given its relatively clear functional semantics, would implementing it as a regular function or member function be better?

Reply:
Consider deleting `get_type_stupport_func_` to simplify the code

Reply:
yes, maybe it's better to implemente it as member function.